idl_parser/OWNERS

```diff
@@ -0,0 +1,6 @@
+bashi@chromium.org
```

Comment:
I think we should remove the file. It doesn't serve any purpose in this repo (it's meant to control access to the code in the chromium repo).

Reply:
Done.